### PR TITLE
Added a line to clarify chmod command

### PR DESCRIPTION
**Pull request by peironggg**
https://piazza.com/class/juazn2axpf35wp?cid=613
Some users faced problem running the command. I think we should add this for clarification.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -322,6 +322,7 @@ Note that there may be some warnings about file times in the future. That is nor
     ```bash
     $ chmod a+x hello.js
     ```
+    NOTE: Do not include '$' when running the command.
```
Comment (angelsl) [inline index.md:326]: Could you add a blank line between this and the closing <code>```</code> please?

**Review by angelsl — CHANGES_REQUESTED** [review]
Thanks! Just one minor thing.

File: index.md (hunk 1 above)
Comment (peironggg) [reply index.md:326]: Updated!

**Comment by angelsl** [comment]
Thanks!

Happy Hacktoberfest :)

**Comment by angelsl** [comment]
By the way, @peironggg, I think your email address is set wrongly in your `git config`. Might wanna check it out.. (doesn't really matter, but it does affect whether commits are linked to your GitHub account.)

**Comment by peironggg** [comment]
> By the way, @peironggg, I think your email address is set wrongly in your `git config`. Might wanna check it out.. (doesn't really matter, but it does affect whether commits are linked to your GitHub account.)

Okay thank you :)

**Pull request merged**